lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -273,7 +278,28 @@ const Reportback = {
         }
       },
     }).done(function(response) {
-      _this.apiUrl = response.meta.pagination.links.next_uri || null;
+      // If response.meta.pagination.links.next, this is from Rogue.
+      if (response.meta.pagination.links.next) {
+        var splitResponseUrl = response.meta.pagination.links.next.split("&");
```

Comment:
So in Rogue, the pagination is structured as 
```
"meta": {
    "pagination": {
      "total": 24,
      "count": 20,
      "per_page": 20,
      "current_page": 1,
      "total_pages": 2,
      "links": {
        "next": "http://rogue.dev:8000/api/v1/reportbacks?page=2"
      }
```
but in Phoenix, it is 
```
"meta": {
    "pagination": {
      "total": 24,
      "count": 20,
      "per_page": 20,
      "current_page": 1,
      "total_pages": 2,
      "links": {
        "next_uri": "http://rogue.dev:8000/api/v1/reportbacks?page=2"
      }
```
I don't think we can change this since we're using `IlluminatePaginatorAdapter`?